lib/tasks/manage.rake

```diff
@@ -23,6 +23,31 @@ namespace :manage do
     end
   end
 
+  def move_to_new_generic_business_finder_emails
+    bulk_business_move_template = <<~BODY.freeze
+      We're updating your subscription to business content related to EU Exit on GOV.UK.
+    BODY
+    business_finder_slug_prefix = "find-eu-exit-guidance-for-your-business-with-sector".freeze
```

Comment:
Nitpick: You could rename this to `old....` so that it's clearer that it's the one being replaced.